### PR TITLE
Add image rename command

### PR DESCRIPTION
**Pull request by roborourke**
This command is a migration tool for making images in the media library compatible with Tachyon.

Addresses #30

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Altis provides a migration command to rename legacy images and make the necessary database updates.
+
+```
+wp media rename-images [--network] [--sites-page=<int>] [--search-replace] [--tables=<tables>] [--include-columns=<columns>]
+```
+
+- `--network` if present will run the process for all sites on the network.
+- `--sites-page` allows you to change the current page of sites. 100 sites will be processed at a time so if you have more you will need to run the command again for each page of sites.
+- `--search-replace` if present will perform a database search and replace process for the updated image names.
```
Comment (jazzsequence) [inline docs/troubleshooting.md:17]: How is this different from the default behavior? Would you pass specific items to search for and replace to this param (e.g. to limit the rename process to a specific batch or naming pattern)? (If so, perhaps a bit of clarity in the docco might be useful along with an example usage.)

**Review by jazzsequence — APPROVED** [review]
LGTM
Left a comment about the `--search-replace` param. I'd like a bit more clarity around that because, afaict, the entire script/cli command is _itself_ a search/replace command, so it's unclear to me what that parameter actually does that's different from the default behavior.

File: docs/troubleshooting.md (hunk 1 above)
Comment (jazzsequence) [reply docs/troubleshooting.md:17]: Or, if the implication is that the database rename doesn't actually happen unless this parameter is passed, that should be noted in the description above.

Comment (roborourke) [reply docs/troubleshooting.md:17]: The images themselves and the attachment data is always updated, the `--search-replace` parameter controls whether or not the rest of the database is checked and updated.

I'll call this out specifically in the preamble.

**Pull request merged**